src/libcore/slice.rs

```diff
@@ -655,6 +655,12 @@ impl<'a, T> Default for &'a [T] {
     fn default() -> &'a [T] { &[] }
 }
 
+#[stable(feature = "mut_slice_default", since = "1.5.0")]
+impl<'a, T> Default for &'a mut [T] {
+    #[stable(feature = "mut_slice_default", since = "1.5.0")]
```

Comment:
@james-darkfox I agree with @Gankro in that this all looks good and is ready to r+ once this is removed. If we do this elsewhere it's just because we were overeager on our annotations and they can be removed later, otherwise may as well try to not add to the problem!